tests/units/cli/test_cli_utils.py

```diff
@@ -13,11 +14,11 @@ class TestUtils:
     Unit Tests for PipelineWise CLI utility functions
     """
     def assert_json_is_invalid(self, schema, invalid_target):
-        """Simple assertion to check if validate function exits with error"""
-        with pytest.raises(SystemExit) as pytest_wrapped_e:
```

Comment:
I'm assuming this is the necessary test changes with the removal of the unreachable code removed in cli/utils.py, right?

Reply:
you are right, the cli/utils.py was updated but correspondent test were left unchanged(fixing legacy cases)